scripts/windows/electron-sign-exe.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
```

Comment:
I was _going_ to say that `electron-sign-exe.sh` is probably too generic of a name, but then I noticed that we also have `electron-sign-app.sh` and `electron-sign-dmg.sh` so this is probably okay ;-)
(but see the comment I just added to the script-renaming PR)